moabb/datasets/physionet_mi.py

```diff
@@ -97,26 +98,31 @@ def __init__(self, imagined=True, executed=False):
 
     def _load_one_run(self, subject, run, preload=True):
         if get_config('MNE_DATASETS_EEGBCI_PATH') is None:
-            set_config('MNE_DATASETS_EEGBCI_PATH',
-                       osp.join(osp.expanduser("~"), "mne_data"))
-        raw_fname = eegbci.load_data(subject, runs=[run], verbose='ERROR',
-                                     base_url=BASE_URL)[0]
+            set_config(
+                'MNE_DATASETS_EEGBCI_PATH', osp.join(osp.expanduser("~"), "mne_data")
+            )
+        raw_fname = eegbci.load_data(
+            subject, runs=[run], verbose='ERROR', base_url=BASE_URL
+        )[0]
         raw = read_raw_edf(raw_fname, preload=preload, verbose='ERROR')
         raw.rename_channels(lambda x: x.strip('.'))
         raw.rename_channels(lambda x: x.upper())
+        # fmt: off
         raw.rename_channels({'AFZ': 'AFz', 'PZ': 'Pz', 'FPZ': 'Fpz',
                              'FCZ': 'FCz', 'FP1': 'Fp1', 'CZ': 'Cz',
                              'OZ': 'Oz', 'POZ': 'POz', 'IZ': 'Iz',
                              'CPZ': 'CPz', 'FP2': 'Fp2', 'FZ': 'Fz'})
+        # fmt: on
```

Comment:
![image](https://user-images.githubusercontent.com/1405370/110359885-58bc5f80-803e-11eb-8e76-6da5732d8ef9.png)

Reply:
maybe
```python
# fmt: off
renames = {
    "AFZ": "AFz", "PZ": "Pz", "FPZ": "Fpz", "FCZ": "FCz", "FP1": "Fp1", "CZ": "Cz",
    "OZ": "Oz", "POZ": "POz", "IZ": "Iz", "CPZ": "CPz", "FP2": "Fp2", "FZ": "Fz",
}
# fmt: on
raw.rename_channels(renames)
```
would be closer to common black style

Reply:
agreed, I added a commit